include/nbl/asset/filters/CMipMapGenerationImageFilter.h

```diff
@@ -96,6 +96,7 @@ class CMipMapGenerationImageFilter : public CImageFilter<CMipMapGenerationImageF
 			for (auto inMipLevel=state->startMipLevel; inMipLevel!=state->endMipLevel; inMipLevel++)
 			{
 				auto blit = buildBlitState(state, inMipLevel);
+				pseudo_base_t::state_type::computePhaseSupportLUT(&blit);
```

Comment:
the call to this should be in `buildBlitState` already

Reply:
Resolved in #343.